### ISSUE TITLE
Google / Analytics360 no longer seems to track member conversions

### ISSUE THREAD

**Issue by ErnieAtLYD** [wherebyus/general-tasks#1429]
via Alexandra: 

Remember when we setup goals in Google so we can track member conversions? I can’t figure out why that’s not working anymore, but Google is over reporting conversions and MailChimp campaign reports aren’t showing any conversions - even for past campaigns that I know had conversions in the past. I wonder if the goals are still setup correctly? And are MailChimp and Google still connected? If you have time/interest in taking a look, too, since nothing was obvious to me.

Campaign > View Report > Analytics360 used to display a conversion no. here that was based on our Google goal of converting to membership, but now for all campaign reports there is no data and it says it hasn’t been setup or its unable to connect:

![image](https://user-images.githubusercontent.com/33945/59381618-bb20e880-8d10-11e9-87ba-11f4c0573bf0.png)

**Comment by ErnieAtLYD** [comment]
I did a couple of things:
 - Update PYS-P and added a [configuration page to the library](http://library.whereby.us/product/growth/pixelyoursite-pro-configuration-settings)
 - Turned off YouTube tracking, which was throwing JS errors
 - Reset Google integration in MailChimp, as I noticed it didn't recognize the integration (see below screenshot).  
 - Turned on Google tracking for a couple of actions inside PYS-P

Before:
![image](https://user-images.githubusercontent.com/33945/59390039-fa593480-8d24-11e9-95c2-bf4e2446d908.png)

After:
![image](https://user-images.githubusercontent.com/33945/59390051-0218d900-8d25-11e9-92ad-696345faefb4.png)


To be honest, I don't know if some, all, or none of these actions will trigger this to work. We'll see.
